stacker_blueprints/elasticache/base.py

```diff
@@ -222,6 +222,11 @@ def create_replication_group(self):
         maintenance_window = variables["PreferredMaintenanceWindow"] or \
             NOVALUE
 
+        if snapshot_window == NOVALUE:
```

Comment:
The problem was snapshot_retention_limit defaults to "0"

The docs say:
`Default: 0 (i.e., automatic backups are disabled for this cache cluster).

`

But this didn't seem to be the case when I tried it. I can have it change the 0 for `SnapshotRetentionLimit` to NOVALUE. Instead of the conditional on `snapshot_window`